src/consume.ts

```diff
@@ -135,12 +143,18 @@ function messageViewerStartPollingCommand(
   const latestResult = os.signal<SimpleConsumeMultiPartitionResponse | null>(null);
   /** Most recent failure info */
   const latestError = os.signal<{ message: string } | null>(null);
-  /** Timestamp of the most recent successful consumption request. */
-  const latestFetch = os.signal<number>(0);
 
-  /** Notify the webview only after flushing the rest of updates. */
+  /** Notify an active webview only after flushing the rest of updates. */
   const notifyUI = () => {
-    queueMicrotask(() => panel.webview.postMessage(["Timestamp", "Success", Date.now()]));
+    queueMicrotask(() => {
+      try {
+        if (panel.visible) {
+          panel.webview.postMessage(["Timestamp", "Success", Date.now()]);
+        }
+      } catch {
+        // panel might be disposed which causes `panel.visible` getter to throw
+      }
```

Comment:
💡 not used to seeing an empty catch block 
Will this throw up to Sentry since it's empty or is it considered "handled" at this point?

Reply:
Just realized there is a cool way to utilize reactive signals here, will address it in the follow up PR.